evalml/tests/automl_tests/test_default_algorithm.py

```diff
@@ -381,3 +383,193 @@ def test_select_cat_cols(
             assert value["columns"] == algo._selected_cols
         elif "Categorical Pipeline - Select Columns Transformer" in component:
             assert value["columns"] == algo._selected_cat_cols
+
+
+@pytest.mark.parametrize(
+    "automl_type",
+    [
+        ProblemTypes.TIME_SERIES_BINARY,
+        ProblemTypes.TIME_SERIES_MULTICLASS,
+        ProblemTypes.TIME_SERIES_REGRESSION,
+    ],
+)
+@patch("evalml.pipelines.components.FeatureSelector.get_names")
+def test_default_algorithm_time_series(
```

Comment:
Once #2867 goes in, any AutoML test that used `IterativeAlgorithm` will use `DefaultAlgorithm` instead. So I won't duplicate coverage for those tests in this PR.